client/state/data-layer/wpcom/sites/utils.js

```diff
@@ -83,7 +84,7 @@ export const dispatchNewCommentRequest = ( dispatch, action, path ) => {
  */
 export const updatePlaceholderComment = ( { dispatch }, { siteId, postId, parentCommentId, placeholderId }, comment ) => {
 	// remove placeholder from state
-	dispatch( { type: COMMENTS_DELETE, siteId, postId, commentId: placeholderId } );
+	dispatch( local( { type: COMMENTS_DELETE, siteId, postId, commentId: placeholderId } ) );
```

Comment:
semantically it should be here still though. I don't want something inadvertently trying to do stuff with it in the future.